apps/events/models/events.py

```diff
@@ -68,6 +70,17 @@ class Event(AbstractBaseModel):
     tags = models.ManyToManyField(
         to=Tag, related_name="events", blank=True)
 
+    # TODO: leave optional?
```

Comment:
I guess so, by design it was optional. If something changes, we still have time to make it required afterwards